instances/matplotlib.colormaps.Greys.jsonld

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This instance seems to be a duplicate.
With the exception of the jsonld name, all the information is identical.

- instances/colormap/matplotlib.colormaps.Greys.jsonld (keep this instance, above)

- instances/matplotlib.colormaps.Greys.jsonld (delete this instnace)

Reply:
deleted. I wondered where this file ended up... the computer froze and then it was gone... I did not realize that I actually saved it in the wrong place :see_no_evil: